docs/user/dmap_old.md

```diff
@@ -0,0 +1,182 @@
+# Reading and Writing DMap format files
+Please see API documentation on [`DmapRead`](code/DmapRead.md) and [`DmapWrite`](code/DmapWrite.md) for information on class methods and attributes. 
+
+Data Map (DMap) is a binary self-describing format that was developed by Rob Barnes. 
+This format is currently the primary format used by SuperDARN. 
+For more information on DMap please see [RST Documentation](https://radar-software-toolkit-rst.readthedocs.io/en/latest/). 
+
+pyDARN provides two classes to read (*DmapRead*) and write (DmapWrite) DMap formated files. 
+
+!!! Note
+    After pyDARN release 1.0, these modules will be moved to it's own python library. 
+
+## DmapRead 
+
+*DmapRead* reads DMap format files and data streams. 
+Here are some examples on how to use the class:
+
+### Read from a file
+
+```python
+import pydarn
+
+dmap_file = "20180220.C0.rkn.fitacf"
+dmap_read = pydarn.DmapRead(fitacf_file)
+dmap_data = dmap_read.read_records()
+```
+
+Accessing data from pyDARN Dmap structure:
+```python
+# loop over every record in the dmap structure
+for dmap_rec in dmap_data:
+  # check if the beam number is 7  
+  if dmap_rec['bmnum'].value == 7:
+    # print the channel number for that record 
+    # corresponding to beam number 7 
+    print(dmap_rec['channel'].value)
+```
+
+To convert to python dictionary:
+```python
+dmap_dict = pydarn.dmap2dict(dmap_data)
+```
+
+Accessing data from a python dictionary:
+```python
+# loop over every record in the dmap structure
+for dmap_rec in dmap_data:
+  # check if the beam number is 7  
+  if dmap_rec['bmnum'] == 7:
+    # print the channel number for that record 
+    # corresponding to beam number 7 
+    print(dmap_rec['channel'])
+```
+
+### Read from a compressed file 
+
+Reading from a `bzip2` compressed file (extension of the file would be *bz2*):
+```python
+import pydarn
+import bz2
+
+dmap_file = ""20170410.1801.00.sas.stream.rawacf.bz2
+
+# does not unzip the actual file but uncompresses it in memory
+with bz2.open(rawacf_stream) as fp:
+    dmap_stream = fp.read()
+
+# compressed data is stored as a data stream, so stream = True
+dmap = pydarn.DmapRead(dmap_stream, True)
+dmap_data = dmap.read_records()
+
+```
+
+### Reading multiple files
+
+Reading multiple DMap formatted files via list:
+```python
+import pydarn 
+import glob 
+
+# Initialize empty DMap data list 
+dmap_list = []
+
+# list of files 
+dmap_files = [""]
```

Comment:
Use glob example here:

```
from glob import glob
dmap_files = glob('path/to/files/*.rawacf')
print(len(dmap_files))
```

Reply:
Ok, seeing the following example, I recommend either adding a comment saying that glob is one way of generating a list or including a list of filenames here.  The empty quotes make it look like you don't need to supply filenames.